cookbooks/steinberger/doc/steinberger.tex

```diff
@@ -0,0 +1,157 @@
+\lstinputlisting[language=prmfile]{cookbooks/steinberger/doc/geometry.part.prm.out}
+Both the top and bottom boundaries allow for free slip. Because the model has periodic side boundary conditions and free slip boundaries at top and bottom, the amount of rigid-body rotation in $\phi$ direction is not constrained. In other words: There is no unique solution. \aspect{} allows it to remove this nullspace from the model (see Section~\ref{sec:nullspace}). Here, we do this by setting the net rotation to zero. 
+\lstinputlisting[language=prmfile]{cookbooks/steinberger/doc/nullspace.part.prm.out}
+The temperature is fixed to 273~K at the top and 3773~K at the bottom boundary. 
+The initial temperature model consists of an adiabatic profile,
+thermal boundary layers at the surface and the core-mantle boundary, 
+and a small harmonic perturbation to initiate convection.
+The gravity profile in the model is based on PREM. 
+
+\paragraph{The equation of state.}
+To use material properties from a temperature--pressure look-up table, we use the Steinberger material model. We have to specify the path to the directory where all the data files we want to use for this model are stored. This includes the files for the viscosity profile, the lateral viscosity variations due to temperature, and all material files containing look-up tables computed by mineral physics software.
+In addition we have to specify the names of these files. 
+In our case, we only have one of these look-up tables, because we only have one composition: pyrolite. 
+But in principle, the material model can use several compositions with one look-up table for each. 
+For intermediate composition values, material properties will then be averaged based on the mass/volume fractions of the individual compositions. 
+
+In addition, there are a few options we can select about how these look-up tables should be used:
+We can decide between interpolating between data points in the lookup table based on the pressure and temperature at the point we need the material properties for, or we can simply take the value from the table that is closest. In our case, we choose the bilinear interpolation because it is more accurate. 
+Second, we can decide how latent heat should be computed: from the thermal expansivity and specific heat, 
+or from the enthalpy (all three properties should be columns in the look-up table). 
+In some cases the look-up table contains the effective thermal expansivity and specific heat, which includes the effect of phase transformations. In a case like that, we simply want to use these values without performing additional latent heat computations because latent heat is already included automatically when using the properties from the look-up table. 
```

Comment:
I don't understand this last half-sentence. The look-up table provides *static* data: for a given p-T pair, what is the density/thermal expansivity/... On the other hand, latent heat is a *dynamic* effect: What happens (i.e., how much heat is released) if you are *moving* through p-T space.

It is ok to ignore this effect, but I think it is *unrelated* to the tabulated properties.